core/services/keystore/vrf_test.go

```diff
@@ -45,32 +47,30 @@ func TestKeyStoreEndToEnd(t *testing.T) {
 	defer cleanup()
 
 	ks := cltest.NewKeyStore(t, store.DB).VRF()
-	// ks := vrf.newVRFKeyStore(vrf.NewORM(store.DB), utils.GetScryptParams(store.Config))
-	key, err := ks.CreateKey(phrase) // NB: Varies from run to run. Shouldn't matter, though
+	ks.Unlock(phrase)
```

Comment:
The API is a bit awkward, here, in that we're calling `Unlock` when there is nothing to actually unlock at the moment.

Reply:
Yeah I agree, but it seemed more important to first get it inline with the other keystores (p2p, ocr, eth) before considering an API change.